operator/config/crd/bases/platform.stackrox.io_centrals.yaml

```diff
@@ -70,6 +70,13 @@ spec:
                           set a password secret; automatically generating a password
                           will not be supported.
                         type: string
+                      enabled:
```

Comment:
`operator/bundle/manifests/platform.stackrox.io_centrals.yaml` should be updated too, I don't know if it's done automatically or you have to do it manually.

Reply:
Thanks, I executed `make bundle` now, that updated the files. Hope this is sufficient.